src/Runner.Listener/Configuration/ConfigurationManager.cs

```diff
@@ -92,9 +92,11 @@ public async Task ConfigureAsync(CommandSettings command)
             _term.WriteSection("Authentication");
             while (true)
             {
-                // Get the URL
-                var inputUrl = command.GetUrl();
-                if (inputUrl.Contains("codedev.ms", StringComparison.OrdinalIgnoreCase))
+                // For testing against a dev deployment of Actions Service, set this environment variable to the deployment URL
+                var actionsServiceTestUrl = Environment.GetEnvironmentVariable("DEV_ACTIONS_SERVICE_TEST_URL");
+                var inputUrl = actionsServiceTestUrl ?? command.GetUrl();
+                if (inputUrl.Contains("codedev.ms", StringComparison.OrdinalIgnoreCase)
+                    || actionsServiceTestUrl != null)
```

Comment:
`DEV_ACTIONS_SERVICE_TEST_URL` might be better set as bool since I assume the L2 test are still going to pass server URL on the commandline args.
```C#
var devActionsService = Environment.GetEnvironmentVariable("DEV_ACTIONS_SERVICE_TEST_URL");
var inputUrl = command.GetUrl();
if (inputUrl.Contains("codedev.ms", StringComparison.OrdinalIgnoreCase) || devActionsService != null)
```

Reply:
Good point, that would be simpler 😆